Readme.md

```diff
@@ -1,29 +1,30 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128532108/13.1.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2313)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [MyObject.cs](./CS/WebSite/App_Code/MyObject.cs) (VB: [MyObject.vb](./VB/WebSite/App_Code/MyObject.vb))
-* [XpoHelper.cs](./CS/WebSite/App_Code/XpoHelper.cs) (VB: [XpoHelper.vb](./VB/WebSite/App_Code/XpoHelper.vb))
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to update a Boolean field using the ASPxCheckBox in a DataItem template
+# Grid View for ASP.NET Web Forms - How to use ASPxCheckBox to update a Boolean field in a DataItem template
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2313/)**
 <!-- run online end -->
 
+This example demonstrates how to use [ASPxCheckBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCheckBox) to implement inline editing for boolean-type fields in [DataItemTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataColumn.DataItemTemplate).
+
+![](checkbox-in-grid.png)
+
+The example uses XPO for demonstration purposes only. To use the example with real databases, modify the [ASPxCallback.Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event handler where update the required record in the "UPDATE" query.
```

Comment:
What do you mean by "where update the required record in the "UPDATE" query"? For example, do you mean:

"where **you** update the required record in the "UPDATE" query"

Reply:
```suggestion
The example uses XPO for demonstration purposes only. To use the example with real databases, modify the [ASPxCallback.Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event handler where you update the required record in the "UPDATE" query.
```